qa/tasks/cephfs/test_admin.py

```diff
@@ -975,6 +975,8 @@ def test_single_path_authorize_on_nonalphanumeric_fsname(self):
         That fs authorize command works on filesystems with names having [_.-] characters
         """
         self.mount_a.umount_wait(require_clean=True)
+        # let's unmount both client before deleting the FS
+        self.mount_b.umount_wait(require_clean=True)
```

Comment:
This makes no sense in `Quincy` branch, because the default `CLIENTS_REQUIRED` is `1`, which is inherit from `CephFSTestCase` class.